app/views/pages/use-cases/twilio-video-integration.liquid

```diff
@@ -0,0 +1,302 @@
+
+#### Create API Key Secret with Curl
+Simply replace **ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX** with your Twilio `sid` and your `auth_token` from the [platformOS Partner Portal](https://portal.apps.near-me.com/): select the Instance your are working on, click Update configuration and then Edit to view the Twilio API Keys.
+<pre class="command-line" data-output="2-50" data-user="user" data-host="host"><code class="language-bash">curl -X POST https://api.twilio.com/2010-04-01/Accounts/ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/Keys.json \ -u ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX:your_auth_token</code></pre>
+    
+We could get the API Secret using many tools but we decided to get it using platformOS. When creating a module, we wanted to do things like create rooms, recordings, compositions and more from inside the context of the module itself. 
+    
+To do this, we needed to connect directly to Twilio from platformOS.
+
+#### Create API Key Secret with platformOS - GraphQL GUI 
+Here, we created an `api_call_notification` and then used the `pos_cli gui sevre <environment>` CLI command to send and receive the response.
+
+1. **Create an API Call Notification**
+
+   To create our **API Key Secret** we send an API call with a request type of `POST` to the URL and with the headers outlined in the [Twilio Identity and Access Management documentation](https://www.twilio.com/docs/iam/keys/api-key-resource).
+
+    #### app/notifications/api_call_notification/twilio_api_secret_create.liquid
+    ```yaml
+    ---
+    name: twilio_api_secret_create
+    format: http
+    headers: |-
+    {%- assign authBasic = context.constants.TWILIO_ACCOUNT_SID | append: ":" | append: context.constants.TWILIO_ACCOUNT_AUTH_TOKEN | base64_encode -%}
+        {
+            "Content-Type": "application/json; charset=utf-8",
+            "Authorization": "Basic {{ authBasic }}"
+        }
+    request_type: POST
+    to: |-
+        https://api.twilio.com/2010-04-01/Accounts/{{ context.constants.TWILIO_ACCOUNT_SID }}/Keys.json
+    ---
+    ```
+
+2. **Send and receive the response**
+
+    To send the **API call notification** and receive the response we used the `pos_cli gui serve <environment>` command. We saw the response returned with our API Key **Secret**.
+
+    ```graphql
+    mutation createSecret {
+        api_call_send(template: {name: "twilio_api_secret_create"}) {
+            response {
+                body
+            }
+        }
+    }
+    ```
+
+#### Fetch API Key resource with platformOS - Form configuration
+
+Here, we were not creating another API Key(Secret) — we were just receiving a representation of the API Key resource that was created. This resource itself is not that helpful, but the process illustrates the workflow when connecting to API resources with a form configuration.
+
+1. **Create a page**
+
+    First, we needed a page to hold a form.
+
```

Comment:
Would be nice to add path to the file just before code starts.

Reply:
Good suggestion. I have added, `#### views/pages/twilio-api.html.liquid`. If this is not what you meant let me know.